### DISCUSSION TITLE
Welcome to TheApiGuys Discussions!

### DISCUSSION THREAD

**Discussion by SagarSingh2003** [SagarSingh2003/TheApiGuys#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SagarSingh2003** [comment]
hey everyone why don't you drop a hi below!!

**Reply by Oscuro05** [reply]
Hi

**Comment by AnsloneKritisch** [comment]
Hi baby <3
